packages/toolkit/src/App.ts

```diff
@@ -69,6 +76,16 @@ class App {
     return this._initPromise;
   }
 
+  public setApiKey(apiKey: string) {
```

Comment:
Is this meant for public use?
This method does not follow return convention on `setCredentials` method of returning a promise, and it doesn't return an error when SQL or CustomStorage is not available.

Reply:
Eh, we could make it protected. It's a method to update the token when it's updated on OAuth apps, there's no point in allowing changing the token from outside.

I didn't return the promise because this has no async consequences, but you're probably right that we should be consistent with the asynchronicity.